lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -10,16 +10,6 @@
  * against the results actually collected from Lighthouse.
  */
 
-const cloneDeep = require('lodash.clonedeep');
-const log = require('lighthouse-logger');
-const LocalConsole = require('./lib/local-console.js');
-
-const NUMBER_REGEXP = /(?:\d|\.)+/.source;
-const OPS_REGEXP = /<=?|>=?|\+\/-|±/.source;
-// An optional number, optional whitespace, an operator, optional whitespace, a number.
-const NUMERICAL_EXPECTATION_REGEXP =
-  new RegExp(`^(${NUMBER_REGEXP})?\\s*(${OPS_REGEXP})\\s*(${NUMBER_REGEXP})$`);
-
 /**
  * @typedef Difference
```

Comment:
To me `typedefs` are constants (type) values. They're also technically exports, due to how tsc treats jsdoc. It definitely seems weird to have them above imports.